### PR TITLE
Higher tolerance for botnet on XPU

### LINKED ISSUE #1305: Models got fail accuracy on BMG but passed on PVC
### 🐛 Describe the bug

```
python benchmarks/dynamo/timm_models.py --accuracy --float16 -d xpu -n10 --training --only botnet26t_256 --backend=inductor
```

Category | Model | Accuracy
-- | -- | --
timm_models_float16_training | botnet26t_256 | fail_accuracy
torchbench_float32_inference | detectron2_maskrcnn_r_50_fpn | fail_accuracy
timm_models_amp_fp16_training | fbnetv3_b | fail_accuracy
timm_models_bfloat16_training | fbnetv3_b | fail_accuracy
timm_models_amp_bf16_training | gernet_l | fail_accuracy
torchbench_float16_training | hf_Longformer | fail_accuracy
torchbench_bfloat16_training | hf_Longformer | fail_accuracy

### Versions

Environments:
Name| Version
-- | --
Device | BMG
OS | Ubuntu 24.10
Driver | 24.45.31740
Torch | v2.6.0-rc5
Torch-xpu-ops | release/2.6
Triton | 3.2.0
Transformers | 243e186efbf7fb93328dd6b34927a4e8c8f24395
Torchvision | 0.21.0+xpu
Torchaudio | 2.6.0+xpu
Torchbench | 766a5e3a189384659fd35a68c3b17b88c761aaac
Timms | ac3470188b914c5d7a5058a7e28b9eb685a62427
Bundle | 2025.0.4.20241205

### PR DESCRIPTION
**Pull request by tbohutyn**
Fixes https://github.com/intel/torch-xpu-ops/issues/1305

Increasing tolerance for botnet26t_256

Analysis showed that this is due to noise on batch_norm. 
In compile we use batch_norm decomposition, if we disable this decomposition - we get results within tolerance. With decomposition enabled, each batch norm's result rmse is ~ 1e-5 which is ulp of float16. This error accumulates and causes bigger differences towards end of the iteration.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @kadeng @chauhang @amjames @Lucaskabela @jataylo

### CODE REVIEW

**Comment by astachowiczhabana** [comment]
Hi @EikanWang
This PR was replaced with https://github.com/pytorch/pytorch/pull/170552 as suggested. Can you please close it (I'm not the author so I don't have permissions)
